conjur/cli.py

```diff
@@ -114,10 +160,19 @@ def run(self, *args):
         if args.verbose is True:
             args.debug = True
 
-        Cli.run_client_action(resource, args)
-
-        # Explicit exit (required for tests)
-        sys.exit(0)
+        # TODO Add tests for exception handling logic
```

Comment:
Is this TODO still needed?

Reply:
yes. Didn't want to add tests for this because touches on variable cli command but still considering it